Gruntfile.js

```diff
@@ -440,7 +440,7 @@ module.exports = function (grunt) {
     grunt.registerTask('unitTest', ['mocha_phantomjs:unitTest']);
 
     //run test
-    grunt.registerTask('runAcceptanceTest', ['exec:firefox']);
+    grunt.registerTask('runAcceptanceTest', ['exec:chrome']);
```

Comment:
This is again a question about our future direction for the website. Do we forsake Firefox testing in favor of chrome? Of course, all of this could become moot if the "website modernization" project, that we have lined up, rips out the entire thing.

Reply:
For reasons discussed, namely Chrome being more popular and its webdriver being significantly more stable, I indeed suggest forsaking Firefox testing, at least for the time being. In the future, when we have time (possibly during the website redesign), we may add Firefox support back.